src/infi/clickhouse_orm/database.py

```diff
@@ -116,6 +117,11 @@ def __init__(self, db_name, db_url='http://localhost:8123/',
             self.readonly = True
         elif autocreate and not self.db_exists:
             self.create_database()
+        self.cluster = cluster
+        if self.cluster: 
+            self.on_cluster_sql = f'ON CLUSTER {cluster}'
```

Comment:
Maybe add a space just in case as having two spaces works, having 0 doesn't
```suggestion
            self.on_cluster_sql = f' ON CLUSTER {cluster}'
```